<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>702</width>
    <height>399</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QSpinBox" name="count_spin">
   <property name="geometry">
    <rect>
     <x>591</x>
     <y>120</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>100000</number>
   </property>
   <property name="value">
    <number>10</number>
   </property>
  </widget>
  <widget class="QPushButton" name="start_btn">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>190</y>
     <width>221</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Start Testing...</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="log_box">
   <property name="geometry">
    <rect>
     <x>14</x>
     <y>270</y>
     <width>671</width>
     <height>121</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="link_input">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>120</y>
     <width>471</width>
     <height>41</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Paste Your Link Here</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
